scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+$nav-height: 90px;
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 4;
+  border-bottom: 2px solid $grey;
+}
+
+.navbar-text {
+  color: $black;
+  font-size: $text-big;
+  padding: 20px;
+}
+
+.navbar-img {
+  height: 90px;
+  padding: 20px;
+  width: 200px;
```

Comment:
Use height OR width

Reply:
Ok, In the next commit there will be only "width"